src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/RichTextBox.xaml

```diff
@@ -0,0 +1,84 @@
+        <Setter Property="VerticalScrollBarVisibility" Value="Auto" />
+        <Setter Property="HorizontalScrollBarVisibility" Value="Disabled" />
+        <Setter Property="MinHeight" Value="34" />
+        <Setter Property="BorderThickness" Value="1" />
+        <Setter Property="Padding" Value="6,4" />
+        <Setter Property="FontSize" Value="14" />
+        <Setter Property="SnapsToDevicePixels" Value="True" />
+        <Setter Property="OverridesDefaultStyle" Value="True" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type RichTextBox}">
+                    <Border
+                        x:Name="MainBorder"
+                        Padding="0"
+                        Background="{TemplateBinding Background}"
+                        BorderBrush="{TemplateBinding BorderBrush}"
+                        BorderThickness="{TemplateBinding BorderThickness}"
+                        CornerRadius="4"
+                        Focusable="False">
+                        <ScrollViewer
+                            x:Name="PART_ContentHost"
+                            Margin="0,0,2,0"
+                            Padding="{TemplateBinding Padding}"
+                            HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+                            VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                            Foreground="{TemplateBinding Foreground}"
+                            HorizontalScrollBarVisibility="{TemplateBinding HorizontalScrollBarVisibility}"
+                            VerticalScrollBarVisibility="{TemplateBinding VerticalScrollBarVisibility}" />
+                    </Border>
+                    <ControlTemplate.Triggers>
+                        <MultiTrigger>
+                            <MultiTrigger.Conditions>
+                                <Condition Property="IsEnabled" Value="True" />
+                                <Condition Property="IsMouseOver" Value="True" />
+                                <Condition Property="IsFocused" Value="False" />
+                            </MultiTrigger.Conditions>
+                            <Setter Property="Background" Value="{DynamicResource TextControlBackgroundPointerOver}" />
+                        </MultiTrigger>
+                        <Trigger Property="IsFocused" Value="True">
+                            <Setter Property="Background" Value="{DynamicResource TextControlBackgroundFocused}" />
+                        </Trigger>
+                    </ControlTemplate.Triggers>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style
+        x:Key="DefaultUiRichTextBoxStyle"
+        BasedOn="{StaticResource DefaultRichTextBoxStyle}"
+        TargetType="{x:Type RichTextBox}">
+
+        <Style.Triggers>
+            <Trigger Property="TextBox.IsTextSelectionEnabled" Value="True">
```

Comment:
This looks backwards to me. I would think you would want to use `Value="False"` here, seeing as the `IsReadOnly` property is then set to `true`.

Reply:
@wjk Thanks for the suggestion, I made changes accordingly. I changed `IsReadOnly` flag to `False`.
If we will set `Value="False"` then it will make `RichTextBox` disabled on load which is not the desired behavior, but by setting `IsReadOnly` to `False` will work as expected.